jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/galleon/s2i_galleon

```diff
@@ -158,6 +162,10 @@ function galleon_reset_mvn_env() {
     if [ -n "$ORIGINAL_MAVEN_ARGS" ]; then
       MAVEN_ARGS=$ORIGINAL_MAVEN_ARGS
```

Comment:
I think this is safer written as:

MAVEN_ARGS=${ORIGINAL_MAVEN_ARGS}

as the args are quite likely to have spaces etc.

Reply:
Thank-you, fixed for both MAVEN_ARGS and MAVEN_ARGS_APPEND